src/library/Uno.Material/Styles/Controls/NavigationView.xaml

```diff
@@ -0,0 +1,1273 @@
+								<VisualState x:Name="PointerOver">
+									<VisualState.Setters>
+										<Setter Target="HoverOverlay.Opacity"
+												Value="1" />
+									</VisualState.Setters>
+								</VisualState>
+								<VisualState x:Name="Pressed">
+									<VisualState.Setters>
+										<Setter Target="PressedOverlay.Opacity"
+												Value="1" />
+									</VisualState.Setters>
+								</VisualState>
+								<VisualState x:Name="Selected">
+									<VisualState.Setters>
+										<Setter Target="LayoutRoot.Background"
+												Value="{StaticResource NavDrawersSelectedColor}" />
+										<Setter Target="ContentPresenter.Foreground"
+												Value="{StaticResource NavDrawersSelectedTextColor}" />
+										<Setter Target="Icon.Foreground"
+												Value="{StaticResource NavDrawersSelectedIconColor}" />
+									</VisualState.Setters>
+								</VisualState>
+								<VisualState x:Name="PointerOverSelected">
+									<VisualState.Setters>
+										<Setter Target="HoverOverlay.Opacity"
+												Value="1" />
+										<Setter Target="LayoutRoot.Background"
+												Value="{StaticResource NavDrawersSelectedColor}" />
+										<Setter Target="ContentPresenter.Foreground"
+												Value="{StaticResource NavDrawersSelectedTextColor}" />
+										<Setter Target="Icon.Foreground"
+												Value="{StaticResource NavDrawersSelectedIconColor}" />
+									</VisualState.Setters>
+								</VisualState>
+								<VisualState x:Name="PressedSelected">
+									<VisualState.Setters>
+										<Setter Target="PressedOverlay.Opacity"
+												Value="1" />
+										<Setter Target="LayoutRoot.Background"
+												Value="{StaticResource NavDrawersSelectedColor}" />
+										<Setter Target="ContentPresenter.Foreground"
+												Value="{StaticResource NavDrawersSelectedTextColor}" />
+										<Setter Target="Icon.Foreground"
+												Value="{StaticResource NavDrawersSelectedIconColor}" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+
+							<VisualStateGroup x:Name="DisabledStates">
+								<VisualState x:Name="Enabled" />
+								<VisualState x:Name="Disabled">
+									<VisualState.Setters>
+										<Setter Target="LayoutRoot.Background"
+												Value="{StaticResource OnSurfaceDisabledBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+
+							<VisualStateGroup x:Name="IconStates">
+								<VisualState x:Name="IconVisible" />
+								<VisualState x:Name="IconCollapsed">
+									<VisualState.Setters>
+										<Setter Target="IconBox.Visibility"
+												Value="Collapsed" />
+										<Setter Target="IconColumn.Width"
+												Value="16" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+						<!-- Wrap SelectionIndicator in a grid so that its offset is 0,0 - this enables the offset animation. -->
```

Comment:
We don't have selection indicator because it is just an overlay right? so we can remove this if we don't need it? What do you think?

Reply:
same uno pasta